src/mdesktopentry.cpp

```diff
@@ -571,5 +573,5 @@ QString MDesktopEntry::url() const
 
 bool MDesktopEntry::isSandboxed() const
 {
-    return d_ptr->keyFile.hasSection(SailjailSection);
+    return value(SailjailSection, SandboxingKey) != DisabledValue && d_ptr->keyFile.hasSection(SailjailSection);
```

Comment:
Looks a lot like: 1st check if value in section is something, then check if said section exists at all - which is a tad confusing. Do both of these refer to the same SailjailSection in the same file?

Reply:
Yeah, they refer to the same section in the same file. The first check must return false if the second check returns false, so it's just a small optimization to have them in this order. Maybe I should just revert the order to make it more readable even if it sometimes costs a few instructions...?

If the file has X-Sailjail section and Sandboxing key in that section doesn't have value Disabled, then the file defines sandboxing, otherwise it doesn't.

Reply:
Swapped the order.